go/teams/rpc_exim.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// Export-Import for RPC for Teams
+
+package teams
+
+import (
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func (t *Team) ExportToTeamPlusAllKeys(idTime keybase1.Time) keybase1.TeamPlusAllKeys {
+	var perTeamKeys = make(map[int]keybase1.PerTeamKey)
+	var err error
+	var i = 0
+	for err == nil {
+		perTeamKeys[i], err = t.Chain.GetPerTeamKeyAtGeneration(i)
+		i++
+	}
+	// hack
+	// for i := 0; i < t.Chain.GetLatestSeqno(); i++ {
```

Comment:
Yeah types like Seqno and Time tend to have different representations in different places, and need to get typecast sometimes. With Time you have to be really careful about units, but with Seqno it's pretty safe.